handbook/people-ops/how-we-engage-talent-outside-the-us/index.md

```diff
@@ -80,7 +80,7 @@ For candidates currently on a visa who prefer to keep their status in a particul
 
 For candidates in the US in need of sponsorship and who have an existing visa, we will accept a visa transfer from the former employer and sponsor the existing visa for the candidate so long as the visa is compatible with an all-remote work setting.
 
-For candidates in need of US sponsorship that require a new visa, we will agree to sponsor new visas for teammates so long as they have worked at Sourcegraph a minimum of 12 consecutive months prior to initiating a visa petition. 
+For candidates in need of US sponsorship, whether that requires initiating a new visa process or a permanent residency (green card) process, Sourcegraph will agree to initiate and sponsor a new visa or green card process for teammates so long as they have worked at Sourcegraph a minimum of 12 consecutive months prior to initiating a the visa or permanent residency petition. 
```

Comment:
```suggestion
For candidates in need of US sponsorship, whether that requires initiating a new visa process or a permanent residency (green card) process, Sourcegraph will agree to initiate and sponsor a new visa or green card process for teammates so long as they have worked at Sourcegraph a minimum of 12 consecutive months prior to initiating the visa or permanent residency petition. 
```

Reply:
What if they have an existing green card process in motion with their former employer?
* Will they have to wait 12 months before resuming the process at Sourcegraph?
* Will they need to restart that process upon switching employers? (Looks like there are some legal considerations there it might be worth flagging: https://www.nolo.com/legal-encyclopedia/can-change-employers-while-green-card-application-pending.html)

Reply:
Just added more language to clarify this a bit further, along with the differences for personal versus role required sponsorship and case-by-case decisions for hard-to-fill roles.